libs/fgdbg/client/src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,85 @@
+import androidx.compose.material.MaterialTheme
```

Comment:
More important: tools should go in tools/, not in libs/

Reply:
Makes sense, I'll do this on a followup PR to keep this one in scope. 
Should I move everything or just `fgdbg/client`?
I am mostly mirroring what `matdbg` does, should `matdbg/web` go inside `tools/` too?